Code/Tools/ProjectManager/Source/EditAGemScreen.cpp

```diff
@@ -107,7 +107,19 @@ namespace O3DE::ProjectManager
         m_originURL->lineEdit()->setText(oldGemInfo.m_originURL);
         m_repositoryURL->lineEdit()->setText(oldGemInfo.m_repoUri);
 
+        HandleGemDetailsTab();
+
         m_oldGemName = oldGemInfo.m_name;
+
+        //load the supported platforms
+        for(int i = GemInfo::NumPlatforms-1; i >= 0; i--)
+        {
+            const GemInfo::Platform platform = static_cast<GemInfo::Platform>(1 << i);
+            if(platform & oldGemInfo.m_platforms)
+            {
+                m_platformOptions->Enable(GemInfo::GetPlatformString(platform));
+            }
+        }
```

Comment:
spacing: There should be a space around the subtraction operator.
However a more foolproof way of iterating backwards is to defer decrementing the index until the array it needs to be accessed.

This makes it so even the index type is an unsigned integer type, then it will still work regardless.

The method currently being used only work for signed types, as unsigned types would result in an infinite loop

```suggestion
        for(int i = GemInfo::NumPlatforms; i > 0; i--)
        {
            const GemInfo::Platform platform = static_cast<GemInfo::Platform>(1 << (i - 1));
            if(platform & oldGemInfo.m_platforms)
            {
                m_platformOptions->Enable(GemInfo::GetPlatformString(platform));
            }
        }
```